node/Vagrantfile

```diff
@@ -0,0 +1,38 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+    # Start with base vagrant box. This one is only 271 MB in size.
+    config.vm.box = "minimal/trusty64"
+
+    # Create forwarding ports for client-guest machine access via localhost.
+    config.vm.network :forwarded_port, guest: 80, host: 7331, auto_correct: true
+    config.vm.network :forwarded_port, guest: 6000, host: 9331, auto_correct: true
+
+    # Add the tty fix as mentioned in issue 1673 on vagrant repo
+    # To avoid 'stdin is not a tty' messages
+    # vagrant provisioning in shell runs bash -l
+    config.vm.provision "fix-no-tty", type: "shell" do |s|
+        s.privileged = false
+        s.inline = "sudo sed -i '/tty/!s/mesg n/tty -s \\&\\& mesg n/' /root/.profile"
+    end
+
+    # avoiding ssh key insert
+    config.ssh.insert_key = false
```

Comment:
I think we should add this to all Vagrantfiles for every stack since without it, sometimes vagrant is not stable and can block access of vagrant ssh simply due to the keys being inserted.